integration/vscode/ada/package.json

```diff
@@ -138,6 +138,23 @@
                         "default": "iso-8859-1",
                         "description": "The character set to use while reading files from disk."
                     },
+                    "ada.displayTypeHierarchy": {
```

Comment:
it should not be called `displayTypeHierarchy`, since this is not really a hierarchy of types. How about `displayMethodAncestryOnNavigation` ? a bit long but we don't care, this is not shown to standard users...

Reply:
Agreed, I will change it